blueprints/app/files/package.json

```diff
@@ -32,8 +32,8 @@
     "<% } %>@ember/optional-features": "^2.1.0",
     "@ember/string": "^3.1.1",
     "@ember/test-helpers": "^3.3.0<% if (embroider) { %>",
-    "@embroider/compat": "^3.5.1",
-    "@embroider/core": "^3.4.10",
+    "@embroider/compat": "^3.5.6",
+    "@embroider/core": "^3.4.14",
     "@embroider/webpack": "^3.2.3<% } %>",
```

Comment:
`@embroider/webpack`  should be v4+

https://github.com/embroider-build/embroider/releases/tag/v3.4.8-%40embroider%2Fcompat

might be a security thing? idk